src/components/vm/hostdevs/hostDevAdd.jsx

```diff
@@ -0,0 +1,236 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from "react";
+import cockpit from "cockpit";
+import {
+    Table,
+    TableBody,
+    TableHeader,
+    TableVariant,
+} from '@patternfly/react-table';
+import {
+    Button,
+    DescriptionList,
+    Form,
+    FormGroup,
+    Modal,
+    Radio
+} from "@patternfly/react-core";
+
+import { ModalError } from "cockpit-components-inline-notification.jsx";
+import { domainAttachHostDevice, domainGet } from "../../../libvirtApi/domain.js";
+import { findHostNodeDevice } from "../../../helpers.js";
+import { getOptionalValue } from "./hostDevCard.jsx";
+import "./hostDevAdd.scss";
+
+const _ = cockpit.gettext;
+
+const TypeRow = ({ idPrefix, type, setType }) => {
+    return (
+        <FormGroup fieldId="usb_device"
+                   label={_("Type")}
+                   isInline
+                   hasNoPaddingTop>
+            <Radio id="usb_device"
+                   name="usb_device"
```

Comment:
`name` in a group should be the same, in this case `type` for example.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#defining_a_radio_group